packages/webos/LS2Request/LS2Request.js

```diff
@@ -24,12 +30,18 @@ export default class LS2Request {
 		onSuccess = null,
 		onFailure = null,
 		onComplete = null,
-		subscribe = false
+		onTimeout = timeoutHandler,
```

Comment:
nit: should it be `onTimeOut` with capital "O"?

Reply:
I think "timeout" is correct.